Modules/@babylonjs/react-native/FontFace.ts

```diff
@@ -0,0 +1,35 @@
+import { Tools } from '@babylonjs/core';
+
+// Declare _native to get access to NativeCanvas.
+declare var _native: any;
+
+/**
+ * Partial Polyfill for FontFace Web API to wrap the NativeCanvas object.
+ */
+export class FontFace {
+  private source: string | ArrayBuffer | undefined;
+  private status: "unloaded" | "loading" | "loaded" | "error" = "unloaded";
+  public constructor(public readonly family: string, source: string | ArrayBuffer) {
+    this.source = source;
+  }
+
+  public async load(): Promise<void> {
+    try {
+      this.status = "loading";
+      if (typeof this.source === 'string') {
+        this.source = await Tools.LoadFileAsync(this.source as string);
```

Comment:
The cast shouldn't be needed as TypeScript will have narrowed it to `string` after the conditional check on line 19.
```suggestion
        this.source = await Tools.LoadFileAsync(this.source);
```